### DISCUSSION TITLE
Postgresql - Data provider date filter not working

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7893] [Help] [answered]
I have a "date_cloture" date field in my postgreSQL table.
If I put a filter "empty", or "not empty" it doesn't work, see :

<img width="702" alt="image" src="https://user-images.githubusercontent.com/28592069/191705362-03745b57-b553-4b03-82d0-f6250809c913.png">

**Comment by NicolasFabre81** [comment, marked as answer]
A working workaround : make a query with the "is not null" SQL filter, and use the query instead of the table in the DP.
But I think it's a bug @melohagan

**Reply by melohagan** [reply]
Thanks Nicolas, I've confirmed the bug and raised it here: https://github.com/Budibase/budibase/issues/7987
